src/main/java/net/mcreator/ui/minecraft/villagers/JVillagerTradeProfession.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2021, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.ui.minecraft.villagers;
+
+import net.mcreator.element.parts.VillagerTradeEntry;
+import net.mcreator.element.types.VillagerTrade;
+import net.mcreator.minecraft.ElementUtil;
+import net.mcreator.ui.MCreator;
+import net.mcreator.ui.component.util.PanelUtils;
+import net.mcreator.ui.init.L10N;
+import net.mcreator.ui.init.UIRES;
+import net.mcreator.workspace.Workspace;
+
+import javax.swing.*;
+import java.awt.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class JVillagerTradeProfession extends JPanel {
```

Comment:
[Change parent class](https://github.com/MCreator/MCreator/commit/0591267c2ae51c3153109a6e690ae6d04b3b0511#diff-2dacfe340fcf5601b93c94d0a1a5155613e6a653e994019374b9c5990a534e6bR36) to `JEntriesList` so that your list and its entries can use ME help context and update code preview generator.

Reply:
Done!